v4l2-request-test.c

```diff
@@ -257,11 +258,34 @@ int main(int argc, char *argv[])
 	setup_config(&config);
 
 	while (1) {
-		opt = getopt(argc, argv, "v:m:d:D:s:f:P:ilqh");
+	        int option_index = 0;
+		static struct option long_options[] = {
+						       {"video_path",  required_argument, 0, 'v' },
+						       {"media_path",  required_argument, 0, 'm' },
+						       {"drm_path",    required_argument, 0, 'd' },
+						       {"drm_driver",  required_argument, 0, 'D' },
+						       {"slice_path",  required_argument, 0, 's'},
+						       {"fps",         required_argument, 0, 'f' },
+						       {"preset_name", required_argument, 0, 'P'},
+						       {"interactive", no_argument,       0, 'i'},
+						       {"loop",        no_argument,       0, 'l'},
+						       {"quiet",       no_argument,       0, 'i'},
+						       {"help",        no_argument,       0, 'h'},
+						       {0,             0,                 0,  0 }
+		};
+
+		opt = getopt_long(argc, argv, "v:m:d:D:s:f:P:ilqh",
+			     long_options, &option_index);
 		if (opt == -1)
 			break;
 
 		switch (opt) {
+		case 0:
```

Comment:
Not sure what this case is for, but it looks like it should be removed.

Reply:
reminescense from example code to handle non string options / option extensions from commandline. Will be thrown out.